src/utilities/bcl/templates/ReportingMeasure/measure.rb

```diff
@@ -24,16 +24,15 @@ def modeler_description
   end
 
   # define the arguments that the user will input
-  def arguments(model)
+  def arguments(model = nil)
     args = OpenStudio::Measure::OSArgumentVector.new
 
-    # model is available to create model-dependent arguments
-    if model.getThermalZones.size > 1
-      add_for_thermal_zones = OpenStudio::Measure::OSArgument.makeBoolArgument('add_for_thermal_zones', true)
-      add_for_thermal_zones.setDisplayName('Add output variables for ThermalZones')
-      add_for_thermal_zones.setDescription('Tests for passing model to arguments() method of ReportingMeasure')
-      args << add_for_thermal_zones
-    end
+    # bool argument to report report_drybulb_temp
```

Comment:
I kinda like the idea of also having an example of using model to create model-dependent arguments.